src/screens/GroupSetting/index.js

```diff
@@ -92,7 +94,7 @@ export default GroupSetting;
 const styles = StyleSheet.create({
   container: {flex: 1, backgroundColor: '#fff'},
   subtitleStyle: {
-    color: COLORS.holyTosca
+    color: COLORS.holytosca
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

There seems to be a change in the color constant name from `COLORS.holyTosca` to `COLORS.holytosca`. Ensure that the new constant exists in the COLORS object and is correctly spelled.



<!-- This is an auto-generated comment by OSS CodeRabbit -->